VERSION_NUMBER

```diff
@@ -1 +1 @@
-1.8.0
+1.8.1
```

Comment:
right now change this to 1.8.150 or something for testing. Once our test packages are tested we can change this back to 1.8.1

Reply:
other places it is ok to keep it 1.8.1... since the package id is determined based on the version number in this file only

Reply:
Thanks for the reminder. I will use 1.8.0.1 because 1.8.150 is higher than 1.8.1